src/course-about/course-intro/course-media/CourseMedia.scss

```diff
@@ -24,12 +24,8 @@
 .course-about-intro-wrapper {
   @media (--pgn-size-breakpoint-max-width-xl) {
     .row {
-      flex-direction: column-reverse;
```

Comment:
[inform]: According to the new changes, we no longer need `column-reverse`, `column` is enough.